frontend/src/libs/packages/notification/notification.package.ts

```diff
@@ -0,0 +1,15 @@
+import { type ToastOptions } from 'react-toastify';
+import { toast } from 'react-toastify';
+
+class Notification {
+  public error(message: string): void {
+    this.show(message, {
+      type: 'error',
+    });
+  }
+  private show(message: string, parameters: ToastOptions): void {
```

Comment:
```suggestion
  private show(message: string, parameters: yourowntype): void {
```
create your own type